mne/bem.py

```diff
@@ -1050,6 +1053,11 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
 
         .. versionadded:: 0.12
 
+    copy : bool
+        If True (default False), use copies instead of symlinks for surfaces
+        (if they do not already exist).
```

Comment:
Usually we write "Defaults to False" right?

Reply:
It varies across the codebase

Reply:
in bem it uses (default bla).